solr/core/src/test/org/apache/solr/TestDistributedSearch.java

```diff
@@ -910,13 +910,11 @@ public void test() throws Exception {
 
     //SOLR 3161 ensure shards.qt=/update fails (anything but search handler really)
     // Also see TestRemoteStreaming#testQtUpdateFails()
-    try {
-      ignoreException("isShard is only acceptable");
-      // query("q","*:*","shards.qt","/update","stream.body","<delete><query>*:*</query></delete>");
-      // fail();
-    } catch (SolrException e) {
-      //expected
-    }
+
```

Comment:
[0] Did you intend to leave this commented out?  (I know this PR is a WIP, so totally possible you're aware of this already.  Just wanted to mention it in-case).

Reply:
Oops my bad, I'll remove it ASAP

Reply:
Yes, since it was commented out beforehand, I just changed the test to use the new syntax, but I figured there was a reason it was commented out.